### PR TITLE
Update project-online-software-boundaries-and-limits.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-ProjectServer/issues/55
Added to note.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: ProjectOnline/project-online-software-boundaries-and-limits.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ To keep Project Online performing at its best, there are some limits to how much
 - ﻿Initial quota of 25GB per Project Web App site. Please see [Tune Project Online performance](tune-project-online-performance.md) for more information.
```
Comment (JohanFreelancer9) [inline ProjectOnline/project-online-software-boundaries-and-limits.md]:
```suggestion
- ﻿Initial quota of 25 GB per Project Web App site. Please see [Tune Project Online performance](tune-project-online-performance.md) for more information. 
```

File: ProjectOnline/project-online-software-boundaries-and-limits.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ To keep Project Online performing at its best, there are some limits to how much
 - ﻿Initial quota of 25GB per Project Web App site. Please see [Tune Project Online performance](tune-project-online-performance.md) for more information. 
     
     > [!NOTE]
-    > The 25GB limit for each Project Online database is separate from the [SharePoint Online limits](https://support.office.com/article/8f34ff47-b749-408b-abc0-b605e1f6d498) where Project Web App is enabled. 
+    > The 25GB limit for each Project Online database is separate from the [SharePoint Online limits](https://support.office.com/article/8f34ff47-b749-408b-abc0-b605e1f6d498) where Project Web App is enabled. The number of Project Web App sites within a tenant is the same as the maximum number of SharePoint Site Collections allowed in a tenant.
```
Comment (JohanFreelancer9) [inline ProjectOnline/project-online-software-boundaries-and-limits.md]:
```suggestion
    > The 25 GB limit for each Project Online database is separate from the [SharePoint Online limits](https://support.office.com/article/8f34ff47-b749-408b-abc0-b605e1f6d498) where Project Web App is enabled. The number of Project Web App sites within a tenant is the same as the maximum number of SharePoint Site Collections allowed in a tenant. 
```

**Comment by scanum** [comment]
@efrene PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@efrene Any updates on this? Thanks!

**Comment by scanum** [comment]
@efrene Can someone else review this? Andrea is no longer with us.

**Pull request merged**